src/rules/store/no-multiple-global-stores.ts

```diff
@@ -16,32 +24,64 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
       category: 'Best Practices',
       description: 'There should only be one global store injected.',
       recommended: 'warn',
+      suggestion: true,
     },
     schema: [],
     messages: {
-      [messageId]: 'Global store should be injected only once.',
+      [noMultipleGlobalStores]: 'Global store should be injected only once.',
+      [noMultipleGlobalStoresSuggest]: 'Remove this reference.',
     },
   },
   defaultOptions: [],
   create: (context) => {
+    const sourceCode = context.getSourceCode()
     const collectedStores = new Set<TSESTree.Identifier>()
 
     return {
       [injectedStore](node: TSESTree.Identifier) {
         collectedStores.add(node)
       },
       [constructorExit]() {
-        if (collectedStores.size > 1) {
-          for (const node of collectedStores) {
-            context.report({
-              node,
-              messageId,
-            })
-          }
+        const stores = [...collectedStores]
+        collectedStores.clear()
+
+        if (stores.length <= 1) {
+          return
         }
 
-        collectedStores.clear()
+        for (const node of stores) {
+          context.report({
+            node,
+            messageId: noMultipleGlobalStores,
+            suggest: [
+              {
+                fix: (fixer) =>
+                  fixer.removeRange(getRemoveRangeFor(node, sourceCode)),
+                messageId: noMultipleGlobalStoresSuggest,
+              },
+            ],
+          })
+        }
       },
     }
   },
 })
+
+function getRemoveRangeFor(
```

Comment:
Oh, this was more code than I had expected 😅

Reply:
Yeah! Dealing with tokens (mostly parenthesis, commas and **comments**) is really, really boring and lead to a verbose code. I wish it were possible to use use selectors for fixer/suggestions as well 😄